Objects/setobject.c

```diff
@@ -757,6 +757,18 @@ frozenset_hash(PyObject *self)
     return hash;
 }
 
+static Py_hash_t
+set_hash(PyObject *self)
```

Comment:
This name is confusing I think =/

Reply:
When refactoring `frozenset_hash()`, I renamed it as `set_hash_func()`. Do you have any other suggestions?

Reply:
Ah I didn't see this comment. I made a separate review with a suggestion.